### ISSUE TITLE
VoiceOver does not speak bottom tab labels, checklist items, dragging tasks, pet feeding bar

### ISSUE THREAD

**Issue by Alys** [HabitRPG/habitrpg-ios#945]
"Hi, I use VoiceOver with habitica. VoiceOver used to work with previous versions of habitica. In the latest update, however, VoiceOver does not speak the tab labels at the bottom, such as        TV ect. I flick left or rite and hear, "tab one of five  ."

"When I create a new daily with several checklist items, VoiceOver no longer reads the expanded checklist items for me to check them. When I double tap on expand checklist and flick up or down, VoiceOver jumps to complete task and delete. I should be able to read each item and double tap to check them off with a one finger double tap."

iOS Version: 13.7
Device: iPhone 6s
App Version: 3.0.3 (520)
User ID: 6c7db23f-52b2-4e56-a0f0-7011df82dfe9

**Comment by shanaqui** [comment]
Another report: "I'm unable to expand or collapse a checklist. I see how to do it, but when I try to expand a checklist, I don't know if it's even working because voiceover isn't reading the expanded list. Again, this worked fine before the new overhaul.  I'm unable therefore to take part in the dungeons and dailies quest or to create any check list." 

Dragging items also does not work: "On todos, for example, if I swipe up I can see "drag 1" or drag 2.  I'm guessing this should be to move up or down?  But that should be clarified."

iOS Version: 14.1
Device: iPhone 11 Pro
App Version: 3.0.3 (520)

**Comment by shanaqui** [comment]
And in addition to that last one: "I am unable to see the percentage of the growth of my pet upon feeding it.  Again this was not a problem for voiceover to read before the visual overhaul.  So I don't know any more what foods will work for each pet, and there's no way for a voiceover user who's new to the game to know either.  I only know some of the foods because Ive been playing for a while."
